scss/utilities/_clearfix.scss

```diff
@@ -1,3 +1,11 @@
 .clearfix {
   @include clearfix();
 }
+
+@each $breakpoint in map-keys($grid-breakpoints) {
+  @include media-breakpoint-only($breakpoint) {
```

Comment:
I was thinking of a picture gallery e.g., where this example might be quite useful.

Look at line 3 of the example
```html
  <div class="clearfix-sm"> <!-- Here we only want to clear on sm, not on md, lg & xl -->
```
here you would write
```html
  <div class="clearfix-sm-up un-clearfix-md-up"> <!-- Here we only want to clear on sm, not on md, lg & xl -->
```

Or line 7 would be:
```html
  <div class="clearfix-sm-up un-clearfix-md-up clearfix-lg-up">
```

I think this is awkward... there are cases where you want to clear just one breakpoint.